### PR TITLE
aos overflow changed

### PR DESCRIPTION
**Pull request by vanshikapandey**

### CODE REVIEW

**Review by sjchoure — CHANGES_REQUESTED** [review]
Add `once:true` property

**Comment by saviourcode** [comment]
@vanshikapandey Don't close this PR, just push the commits to the same branch

File: script.js [hunk 1]
```diff
@@ -86,3 +86,8 @@ function toggleDarkLight() {
 }
 
 var tilt = $('.js-tilt').tilt();
+
+//aos
+AOS.init({
+  once:false,
```
Comment (saviourcode) [inline script.js:92]: `once:true`

**Review by saviourcode — CHANGES_REQUESTED** [review]

**Review by sjchoure — APPROVED** [review]
LGTM! @vanshikapandey Awesome work!!

**Comment by vanshikapandey** [comment]
Thank you sir :)

**Pull request merged**